lib/memo_wise.rb

```diff
@@ -341,7 +345,10 @@ def #{method_name}
             end
           END_OF_METHOD
         else
-          if MemoWise.has_only_required_args?(method)
+          if MemoWise.has_only_keyword_required_args?(method)
```

Comment:
Just a note—I anticipate this would not be more efficient than the single-arg case. Thoughts on only doing this if it's not a single-`:keyreq` case?